src/services/base/organization.ts

```diff
@@ -0,0 +1,64 @@
+import { FetchType, jsonFetchWrapper } from "@/submodules/javascript-functions/basic-fetch";
+import { BACKEND_BASE_URI } from "./_settings";
+import { convertCamelToSnakeCase } from "@/submodules/javascript-functions/case-types-parser";
+
+export const organizationEndpoint = `${BACKEND_BASE_URI}/api/v1/organization`;
+
+export function getOrganization(onResult: (result: any) => void) {
+    const finalUrl = `${organizationEndpoint}`;
+    jsonFetchWrapper(finalUrl, FetchType.GET, onResult);
+}
+
+export function getOrganizationUsers(onResult: (result: any) => void) {
+    const finalUrl = `${organizationEndpoint}/all-users`;
+    jsonFetchWrapper(finalUrl, FetchType.GET, onResult);
+}
+
+export function getUserInfo(onResult: (result: any) => void) {
+    const finalUrl = `${organizationEndpoint}/user-info`;
+    jsonFetchWrapper(finalUrl, FetchType.GET, onResult);
+}
+
+export function getOverviewStats(onResult: (result: any) => void) {
+    const finalUrl = `${organizationEndpoint}/overview-stats`;
+    jsonFetchWrapper(finalUrl, FetchType.GET, onResult);
+}
+
+export function getAllActiveAdminMessages(onResult: (result: any) => void) {
+    const finalUrl = `${organizationEndpoint}/all-active-admin-messages`;
+    jsonFetchWrapper(finalUrl, FetchType.GET, onResult);
+}
+
+export function getAllUsersWithRecordCount(projectId: string, onResult: (result: any) => void) {
```

Comment:
this is used for the data browser, i think we can moved it there, but nothing critical, can also stay here

Reply:
i checked the BE, it's part of the organization manager, so we can leave it in the organization route